Makefile.common

```diff
@@ -120,8 +126,13 @@ common-docker-tag-latest:
 	docker tag "$(DOCKER_REPO)/$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)" "$(DOCKER_REPO)/$(DOCKER_IMAGE_NAME):latest"
 
 .PHONY: promu
-promu:
-	GOOS= GOARCH= $(GO) get -u github.com/prometheus/promu
+promu: $(PROMU)
+
+.PHONY: $(PROMU)
```

Comment:
@SuperQ Looks like the curl is done for each platforms during the crossbuild. Maybe because of this line ?

Reply:
Yup, that was it. Fixed.